unity-renderer/Assets/Scripts/MainScripts/DCL/WorldRuntime/ParcelScene.cs

```diff
@@ -336,7 +336,18 @@ private void OnDrawGizmosSelected()
             }
         }
 
-        public IDCLEntity GetEntityById(string entityId) { throw new System.NotImplementedException(); }
+        public IDCLEntity GetEntityById(string entityId)
```

Comment:
we are deprecating the string usage for entities and scenes. Can you give it a look where this `GetEntityById` method is called and try to use the long instead? It's probably not much more difficult since before it wasnt even implemented.

Reply:
Thanks, fixed!